src/components/account/OrderScreen.js

```diff
@@ -34,7 +33,7 @@ class OrderScreen extends Component {
 
   renderItem = (item) => {
     const { code, text, row, imageStyle, container } = styles;
-    const currency = priceSignByCode(this.props.navigation.state.params.item.order_currency_code);
+    const { currencySymbol } = this.props;
```

Comment:
Again instead of using order_currency_code to determine currency symbol I am directly using currency symbol that I got from API.